test/lorenz_delta.sh

```diff
@@ -0,0 +1,165 @@
+   $scriptname.dir. This test passes if $scriptname.err is empty.
+
+   Example usage: ./test.sh $0 
+
+EOF
+      exit
+      ;;
+esac
+
+# Determine csplit and mpirun command for this machine 
+OS=`uname`
+case $OS in
+   Linux*) 
+      MACHINES_FILE="hostname"
+      if [ ! -f $MACHINES_FILE ] ; then
+         hostname > $MACHINES_FILE
+      fi
+      RunString="mpirun -machinefile $MACHINES_FILE $*"
+      csplitcommand="csplit"
+      ;;
+   Darwin*)
+      csplitcommand="gcsplit"
+      RunString="mpirun --hostfile ~/.machinefile_mac"
+      ;;
+   *)
+      RunString="mpirun"
+      csplitcommand="csplit"
+      ;;
+esac
+
+
+
+# Setup
+example_dir="../examples"
+driver_dir="../drivers"
+test_dir=`pwd`
+output_dir=`pwd`/$scriptname.dir
+rm -fr $output_dir
+mkdir -p $output_dir
+
+
+# compile the regression test drivers 
+echo "Compiling regression test drivers"
+cd $example_dir
+make clean
+make ex-07
+cd $test_dir
+
+# Run the following regression tests 
+TESTS=( "$RunString -np 1 $example_dir/ex-07 -test" \
+        "$RunString -np 2 $example_dir/ex-07 -tstop 0.5 -ntime 256 -ml 2 -cf 2 -tol 1e-6 -rank 0" \
+        "$RunString -np 2 $example_dir/ex-07 -tstop 0.5 -ntime 256 -ml 2 -cf 2 -tol 1e-6 -rank 1" \
+        "$RunString -np 2 $example_dir/ex-07 -tstop 0.5 -ntime 256 -ml 2 -cf 2 -tol 1e-6 -rank 2" \
+        "$RunString -np 2 $example_dir/ex-07 -tstop 0.5 -ntime 256 -ml 2 -cf 2 -tol 1e-6 -rank 3" \
+        "$RunString -np 2 $example_dir/ex-07 -tstop 0.5 -ntime 256 -ml 3 -cf 2 -tol 1e-6 -rank 0" \
+        "$RunString -np 2 $example_dir/ex-07 -tstop 0.5 -ntime 256 -ml 3 -cf 2 -tol 1e-6 -rank 1" \
+        "$RunString -np 2 $example_dir/ex-07 -tstop 0.5 -ntime 256 -ml 3 -cf 2 -tol 1e-6 -rank 2" \
+        "$RunString -np 2 $example_dir/ex-07 -tstop 0.5 -ntime 256 -ml 3 -cf 2 -tol 1e-6 -rank 3" \
+        "$RunString -np 2 $example_dir/ex-07 -tstop 0.5 -ntime 256 -ml 3 -cf 2 -tol 1e-6 -rank 1 -defer-lvl 1" \
+        "$RunString -np 2 $example_dir/ex-07 -tstop 0.5 -ntime 256 -ml 3 -cf 2 -tol 1e-6 -rank 2 -defer-lvl 1" \
+        "$RunString -np 2 $example_dir/ex-07 -tstop 0.5 -ntime 256 -ml 3 -cf 2 -tol 1e-6 -rank 3 -defer-lvl 1" \
+        "$RunString -np 2 $example_dir/ex-07 -tstop 0.5 -ntime 256 -ml 3 -cf 2 -tol 1e-6 -rank 1 -defer-iter 1" \
+        "$RunString -np 2 $example_dir/ex-07 -tstop 0.5 -ntime 256 -ml 3 -cf 2 -tol 1e-6 -rank 2 -defer-iter 1" \
+        "$RunString -np 2 $example_dir/ex-07 -tstop 0.5 -ntime 256 -ml 3 -cf 2 -tol 1e-6 -rank 3 -defer-iter 1" \
+        "$RunString -np 2 $example_dir/ex-07 -tstop 0.5 -ntime 256 -ml 3 -cf 2 -tol 1e-6 -rank 1 -nu 1" \
+        "$RunString -np 2 $example_dir/ex-07 -tstop 0.5 -ntime 256 -ml 3 -cf 2 -tol 1e-6 -rank 2 -nu 1 -fcfLyap -noLyap" \
+        "$RunString -np 2 $example_dir/ex-07 -tstop 0.5 -ntime 256 -ml 3 -cf 2 -tol 1e-6 -rank 3 -noLyap" \
+        "$RunString -np 2 $example_dir/ex-07 -tstop 0.5 -ntime 256 -ml 3 -cf 2 -tol 1e-6 -rank 3 -noLyap -fcfLyap" )
```

Comment:
it's often good to ahve one test that verifies the output is insensitive to numprocs ... it's not a must, though